### DISCUSSION TITLE
Applet Install

### DISCUSSION THREAD

**Discussion by listsurfer** [linuxmint/discussions#486] [Questions]
[1]  Where are the required settings located for installing Cinnamon applets? I don't see them in the latest Mint Settings.
[2]  How can I reduce the steps to shutting down? The icon on desktop leads to a password or must use the upper right corner. Too many steps?

**Comment by rcalixte** [comment]
> [1] Where are the required settings located for installing Cinnamon applets? I don't see them in the latest Mint Settings.

Under Settings, you should be able to select Applets. (You can also right-click on a panel and select Applets.) Not all applets have settings, nor are they required, but those that do will have a selectable button next to them if they are enabled like so:

![Applets_001](https://github.com/linuxmint/discussions/assets/10281587/0f702cd8-959b-48eb-83bb-d182fa3bccf8)





> [2] How can I reduce the steps to shutting down? The icon on desktop leads to a password or must use the upper right corner. Too many steps?

I'm not entirely certain what you mean here but depending on how your system is configured, you can use a number of methods to minimize the number of steps to shutting down. [Here is a YouTube video](https://youtu.be/Bm_n-9nhRYU) that goes into depth on various options you can setup on your system.
